src/commands.ts

```diff
@@ -52,17 +58,70 @@ function bzlMakeQueryQuickPickItem(queryItem: BazelQueryItem): BazelQueryQuickPi
 }
 
 async function bzlQuickPickQuery(query: string = '...', options?: QuickPickOptions) {
+
     return Window.showQuickPick(bzlQuery(query).then(deps => {
         return deps.map(bzlMakeQueryQuickPickItem);
-    }, err => {
-        Window.showQuickPick([], {
+    }).catch(async err => {
+        await Window.showQuickPick([], {
```

Comment:
Why do we show another (empty) QuickPick in case of an error? Is it just for showing the `<ERROR>` string as the placeHolder?

Reply:
> Is it just for showing the <ERROR> string as the placeHolder?
Preceisely, it overwrites the existing one. I didn't see documentation on how to close the quickpick programatically. I thought showing there is an error in the placeholder would be better than leaving it there with nothing.